### DISCUSSION TITLE
esp32/Arduino/12.esp32-arduino-lcd1602

### DISCUSSION THREAD

**Discussion by giscus** [bigrich-luo/DocsGiscus#35] [General]
# esp32/Arduino/12.esp32-arduino-lcd1602

https://docs.geeksman.com/esp32/Arduino/12.esp32-arduino-lcd1602.html

**Comment by laichun** [comment]
LiquidCrystal_I2C.h 和 LiquidCrystal_I2C.cpp  这2个文档在哪里呢？

**Reply by CardiganQ** [reply]
我也找不到，哈哈哈，我甚至买的都不是这个板子，不过这些针脚什么的都一一对应着

**Comment by laichun** [comment]
找到啦

**Comment by yf-no-signals** [comment]
嘿嘿
